flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -1372,6 +1381,12 @@
 	/** The environment variable name which contains the location of the lib folder */
 	public static final String ENV_FLINK_LIB_DIR = "FLINK_LIB_DIR";
 
+	/** The environment variable name which contains the location of the bin directory */
+	public static final String ENV_FLINK_BIN_DIR = "FLINK_BIN_DIR";
+
+	/** The environment variable name which contains the Flink installation root directory */
+	public static final String ENV_FLINK_HOME_DIR = "FLINK_HOME";
+
```

Comment:
you sure these should be config values? What if someone wants to set these to `$MESOS_SANDBOX`, then we need to rely on shell expansion on an already started container^^ (Not sure this is a usecase)